### PR TITLE
Added logger system.

### PR DESCRIPTION
**Pull request by iBubbleGun**
Logger system added.
Please, try it.

### CODE REVIEW

File: src/main/java/edu/hillel/homework/logger_system/log4hillel/config_loader/ConsoleLoggerConfigLoader.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+                switch (key) {
+                    case "CONSOLE-LOGGING-ENABLED" -> {
+                        try {
+                            consoleLoggingEnable = Boolean.parseBoolean(value);
+                        } catch (Exception e) {
+                            throw new RuntimeException("Invalid configuration file format.", e);
+                        }
+                    }
+                    case "LOG-LEVEL" -> logLevel = LogLevel.valueOf(value);
+                    case "FORMAT" -> format = value;
```
Comment (kartius) [inline src/main/java/edu/hillel/homework/logger_system/log4hillel/config_loader/ConsoleLoggerConfigLoader.java]: Will be more beautiful to use separated methods in switch-case block

File: src/main/java/edu/hillel/homework/logger_system/log4hillel/config_loader/DatabaseLoggerConfigLoader.java [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+                }
+
+                String key = parts[0].trim().toUpperCase();
+                String value = parts[1].trim();
+
+                switch (key) {
+                    case "DATABASE-LOGGING-ENABLED" -> {
+                        try {
+                            databaseLoggingEnable = Boolean.parseBoolean(value);
+                        } catch (Exception e) {
```
Comment (kartius) [inline src/main/java/edu/hillel/homework/logger_system/log4hillel/config_loader/DatabaseLoggerConfigLoader.java]: The same here and you will avoid a code duplication

**Review by kartius — APPROVED** [review]

File: src/main/java/edu/hillel/homework/logger_system/log4hillel/config_loader/ConsoleLoggerConfigLoader.java (hunk 1 above)
Comment (iBubbleGun) [reply src/main/java/edu/hillel/homework/logger_system/log4hillel/config_loader/ConsoleLoggerConfigLoader.java]: Well done.

File: src/main/java/edu/hillel/homework/logger_system/log4hillel/config_loader/DatabaseLoggerConfigLoader.java (hunk 2 above)
Comment (iBubbleGun) [reply src/main/java/edu/hillel/homework/logger_system/log4hillel/config_loader/DatabaseLoggerConfigLoader.java]: Well done.

**Pull request merged**